Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAccountProtectionPolicy/MSFT_IntuneAccountProtectionPolicy.psm1

```diff
@@ -144,7 +144,7 @@ function Get-TargetResource
     {
         #Retrieve policy general settings
 
-        $policy = Get-MgBetaDeviceManagementIntent -DeviceManagementIntentId $Identity -ErrorAction SilentlyContinue
+        $policy = Get-MgBetaDeviceManagementIntent -DeviceManagementIntentId $Identity -ExpandProperty settings,assignments -ErrorAction SilentlyContinue
```

Comment:
Just as a check:
A lot of other resources have the Group.Read.All permissions added. This resource and the IntuneAntivirusPolicyWindows10SettingCatalog don't have that. Is that correct?